src/connectors/configure-related-items/connectConfigureRelatedItems.ts

```diff
@@ -66,10 +54,7 @@ function createOptionalFilter({
   return `${attributeName}:${attributeValue}<score=${attributeScore || 1}>`;
 }
 
-const connectConfigureRelatedItems: ConfigureRelatedItemsConnector = (
-  renderFn,
-  unmountFn
-) => {
+export default (function connectConfigureRelatedItems(renderFn, unmountFn) {
```

Comment:
Note for other reviewers: the render function is now required for consistency. If it causes trouble, we can create an alternative connector type that makes this render function optional.